openreview/tools.py

```diff
@@ -1798,11 +1804,10 @@ def get_neurips_profile_info(profile, n_years=3):
 
     ## Emails section
     for email in profile.content['emails']:
+        if email.startswith("****@"):
+            raise openreview.OpenReviewException("You do not have the required permissions as some emails are obfuscated. Please login with the correct account or contact support.")
         emails.add(email)
-    ## if institution section is empty, add email domains
-    if not domains:
```

Comment:
this implementation was added by NeurIPS Organizers, I'm not sure why you want to remove the if here?

The comment says if there is not institution listed then use the email as a domain conflict.

Reply:
I combined both of them as I was trying to remove email-email conflict computation. I don't think it will change things either way, but I can revert it.

Reply:
where do you combine them?

Could you write a test for this scenario? using a profile with an email address but no history

Reply:
it's updated [here](https://github.com/openreview/openreview-py/blob/cd624bbb5795a779fba067d1a8b4408df3f4b3dd/openreview/tools.py#L1806). It's same as the regular `get_profiles_info` function.